CONTRIBUTING.md

```diff
@@ -1,18 +1,312 @@
 # Contributing to Conjur
 
-Thanks for your interest in Conjur. Before contributing, please take a moment to read and sign our <a href="https://github.com/cyberark/conjur/blob/master/Contributing_OSS/CyberArk_Open_Source_Contributor_Agreement.pdf" download="conjur_contributor_agreement">Contributor Agreement</a>. This provides patent protection for all Conjur users and allows CyberArk to enforce its license terms. Please email a signed copy to <a href="oss@cyberark.com">oss@cyberark.com</a>.
+Thanks for your interest in Conjur. Before contributing, please take a moment to
+read and sign our <a href="https://github.com/cyberark/conjur/blob/master/Contributing_OSS/CyberArk_Open_Source_Contributor_Agreement.pdf" download="conjur_contributor_agreement">Contributor Agreement</a>.
+This provides patent protection for all Conjur users and allows CyberArk to enforce
+its license terms. Please email a signed copy to <a href="oss@cyberark.com">oss@cyberark.com</a>.
 
 For general contribution and community guidelines, please see the [community repo](https://github.com/cyberark/community).
 
+## Prerequisites
+
+Before getting started, you should install some developer tools. These are not
+required to deploy Conjur but they will let you develop using a standardized,
+expertly configured environment.
+
+1. [git][get-git] to manage source code
+1. [Docker][get-docker] to manage dependencies and runtime environments
+1. [Docker Compose][get-docker-compose] to orchestrate Docker environments
+
+[get-docker]: https://docs.docker.com/engine/installation
+[get-git]: https://git-scm.com/downloads
+[get-docker-compose]: https://docs.docker.com/compose/install
+
+## Build Conjur as a Docker image
+
+It's easy to get started with Conjur and Docker:
+
+1. install dependencies (as above)
+1. clone this repository
+1. run the build script in your terminal:
```

Comment:
These should all start capitalized

Reply:
fixed - even though I didn't write this, you're right, and might as well fix it now